crowbar_framework/app/models/backup.rb

```diff
@@ -167,6 +171,7 @@ def save_archive
     self.version = meta["version"]
     self.size = path.size
     self.created_at = Time.zone.parse(meta["created_at"])
+    self.migration_level = meta["migration_level"]
```

Comment:
Ouch, this means that we take the risk that a backup restore / upgrade will fail.